ext/reflection/tests/ReflectionFunction_hasParameter.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+ReflectionFunction::hasParameter()
+--FILE--
+<?php
+function foo(string $bar) {}
+
+$function = new ReflectionFunction('sort');
+var_dump($function->hasParameter('array'));
+var_dump($function->hasParameter('Array'));
+var_dump($function->hasParameter('string'));
+var_dump($function->hasParameter(0));
+var_dump($function->hasParameter(1));
+var_dump($function->hasParameter(-1));
```

Comment:
Per the other review comment, this should be a ValueError thus either moved to another test file with all the exceptions or just try/catch here